src/pages/components/select-native.mdx

```diff
@@ -105,34 +105,60 @@ import ValidationCode from '!!raw-loader!../../examples/forms/select/Validation.
 
 ## API
 
+The native Select component follows this structure:
+
+```jsx
+<Field>
+  <Label />
+  <Hint />
+  <Select>
+    <option />
+    <option />
+    {/* etc. */}
+  </Select>
+</Field>
+```
+
 ### Field
 
 <Component components={props.data.mdx.components} componentName="field" />
 
+A Field provides accessibility attributes to the child [Select](#select) field by associating it
+with the corresponding [Label](#label) and [Hint](#hint).
+
 <PropSheet components={props.data.mdx.components} componentName="field" />
 
 ### Hint
 
 <Component components={props.data.mdx.components} componentName="hint" />
 
+Nest a Hint within a [Field](#field) component.
+
 <PropSheet components={props.data.mdx.components} componentName="hint" />
 
 ### Label
 
 <Component components={props.data.mdx.components} componentName="label" />
 
+Nest a Label within a [Field](#field) component.
+
 <PropSheet components={props.data.mdx.components} componentName="label" />
 
 ### Message
 
 <Component components={props.data.mdx.components} componentName="message" />
 
+A Message can be used to show validation messages. Nest it within a [Field](#field) component.
```

Comment:
Your previous [Message docs](https://garden.zendesk.com/components/input#message) read a bit better. This feels redundant "A Message can show messages"

Reply:
Updated!